src/main/resources/application.yml

```diff
@@ -74,6 +72,16 @@ workspace:
     username: ${env.db.ws.user}
 
 terra.common:
+  kubernetes:
+    in-kubernetes: true
+  stairway:
+    cluster-name-suffix: workspace-stairway
+    force-clean-start: false # ${env.db.init}
+    max-parallel-flights: 50
+    migrate-upgrade: false # true
```

Comment:
I feel this should be true. But I can not make the call for WSM. @ddietterich

Reply:
Yes, we should upgrade the schema by default.